forum/qa-plugin/q2a-comment-voting-master/qa-comment-voting-layer.php

```diff
@@ -168,13 +168,15 @@ function q_view($q_view)
 		function c_item_main($c_item)
 		{
 			global $topage;
-			if(qa_opt('voting_on_cs') && is_array($this->comment_votes) && isset($c_item['content']) && !isset($c_item['url']) && !strpos($c_item['content'],'question-closed-message')) {
+			if(qa_opt('voting_on_cs') && isset($c_item['content']) && !isset($c_item['url']) && !strpos($c_item['content'],'question-closed-message')) {
 				$vote=0;
 				$flag=0;
-				foreach($this->comment_votes as $vote) {
-					if($vote['postid'] == $c_item['raw']['postid']) {
-						$vote = (int)$vote['vote'];
-						break;
+				if(is_array($this->comment_votes)) {

```

Comment:
Bez tego użytkownik niezalogowany nie dostanie votów i formatowanie będzie niepoprawne. Nie pamiętam zbytnio, czy to jedyny problem z tym kodem, ale robiłem to na samym początku i wyleciało mi z głowy ;)